shared/messagestore/src/main/kotlin/company/thewebhook/messagestore/consumer/PulsarConsumerImpl.kt

```diff
@@ -87,7 +88,7 @@ class PulsarConsumerImpl(
                 val records = consumer.batchReceive()
                 unacknowledgedMessages.addAll(records.map { r -> r.messageId })
                 records
-                    .map { Record(it.topicName, it.value) }
+                    .map { Record(it.topicName, it.value, it.messageId.toByteArray().toString(Charsets.UTF_8)) }
```

Comment:
I searched online - seems like we can just do `String()` instead of `.toString(charsets.UTF_8)`.

Also, is this a safe string? Can you send some instances of this message id as a string?